### PR TITLE
fixed wordCount

### PR DESCRIPTION
**Pull request by amichaelyu**
The regex for the current wordCount is implemented incorrectly.

### CODE REVIEW

**Comment by amichaelyu** [comment]
@connorff

File: src/FormPage/FormPage.tsx [hunk 1]
```diff
@@ -119,7 +119,7 @@ const TextareaWordCountingWidget = (props) => {
   } = props;
 
   const showWordCount = !!schema.word_count;
-  const wordCount = value ? value.split(/\s+/g).length : 0;
+  const wordCount = value.split(/[\s]+/).filter(function (el) {return el !== '';}).length;
```
Comment (connorff) [inline src/FormPage/FormPage.tsx]: Sometimes `value` is undefined (user hasn't filled out a field), so we still need the ternary check.

Also, for the `filter` function, we could rewrite it as `.filter(el => el !== '')` to reduce the line length.

File: src/FormPage/FormPage.tsx [hunk 2]
```diff
@@ -183,7 +183,7 @@ function validate(formData, errors, schema) {
     .filter((key) => !!schema.properties[key].word_count)
     .forEach((key) => {
       const wordCount = schema.properties[key].word_count;
-      if (formData[key] && formData[key].split(/\s+/g).length > wordCount) {
+      if (formData[key] && formData[key].split(/[\s]+/).filter(function (el) {return el !== '';}).length > wordCount) {
```
Comment (connorff) [inline src/FormPage/FormPage.tsx]: Same thing in this `filter`: `.filter(el => el !== '')`

**Review by connorff — CHANGES_REQUESTED** [review]
Hey Michael, thank you for this pull request! Just some minor comments but, again, appreciate it :)

**Review by connorff — APPROVED** [review]
Thanks!

**Pull request merged**